contracts/utils/PausableAdmin.sol

```diff
@@ -133,13 +159,4 @@ contract PausableAdmin is PendingOwnable, Pausable, IPausableAdmin {
             revert PausableAdmin__AddressIsNotPauseAdmin(_admin);
         emit PauseAdminRemoved(msg.sender, _admin);
     }
-
-    /**
-     * @notice Internal function to revoke the role of an admin
-     */
-    function _revokePauseAdmin(address _revokedAdmin) internal {
-        if (!_pauseAdmins.remove(_revokedAdmin))
-            revert PausableAdmin__AddressIsNotPauseAdmin(_revokedAdmin);
-        emit PauseAdminRevoked(_revokedAdmin);
-    }
```

Comment:
Removed this part as OZ does it that way (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/AccessControl.sol#L179-L183)